### ISSUE TITLE
Is there a option to disable generating CSS inside <style> tag ?

### ISSUE THREAD

**Issue by fsasvari** [jjriv/emogrifier#353]
Is there a option to disabled generating CSS inside `<style>` tag ?

I'm sending HTML and CSS code to emogrifier.

`$emogrifier = new Emogrifier($html, $css);`

In HTML i already have `<style>` tag with fixed manually added styles. In CSS I have all CSS that needed to be inlined.

The problem is if I use `$emogrifier->disableStyleBlocksParsing();` it leaves my original <style> but adds 2 other ? How is that  possible ?

**Comment by oliverklee** [comment]
Hi @fsasvari, thanks for reporting this issue.

If you do not want you styles to be inlined, why are you using Emogrifier? (The point of Emogrifier is to inline the CSS. If you do not need this, there is no point in using Emogrifier.)

But maybe I have completely misunderstood the issue. Could you please post/attach the input HTML and CSS, the actual output and output you would have expected? Thanks!

**Comment by fsasvari** [comment]
@oliverklee No, i want to inline my styles, but the problem is that my original `<style>` data is being truncated.

Is there a option that will inline data from external CSS, but leave my `<style>` tag intact ?

I'm using ZURB foundation for emails, and it has some custom CSS for Outlook Email client like:

```
#outlook a {
  padding: 0;
}
.ExternalClass {
  width: 100%; }
  .ExternalClass,
  .ExternalClass p,
  .ExternalClass span,
  .ExternalClass font,
  .ExternalClass td,
  .ExternalClass div {
    line-height: 100%;
}
```
In HTML there is no `#outlook` and no `.ExternalClass` so Emogrifier just exclude does classes and truncates them from `<style>` tag.

**Comment by oliverklee** [comment]
Actually, disableInlineStylesParsing should do the trick. It should not add additional style nodes, though. If it does so, this is a bug. Could you please create a ticket with the input HTML + CSS, the actual result and the expected result? Most helpful would be code that is just enough to reproduce the problem, but is minimal in all other aspects.

**Comment by fsasvari** [comment]
Actually, I'm using [https://github.com/petecoop/laravel-inky](petecoop/laravel-inky) which generates HTML from inky templates. But I don't think there is a problem with laravel-inky class.

On line 33 at https://github.com/petecoop/laravel-inky/blob/master/src/InkyCompilerEngine.php#L33 there is:

`$emogrifier = new Emogrifier($results, $styles);`

when I use `dump($styles)` everything is ok, I get whole CSS file inside, but Emogrifier strips not used styles and truncated them from `<style>` tag.

I added one line after:

`$emogrifier->disableStyleBlocksParsing();`

But Emogrifier adds 2 more `<style>` tags, don't know why ?

I just tested a bit more... I think it's problem with Laravel blade and `@media only screen`. I generate CSS, puts them as blade templates and includes them inside `<style>` tag

```
<style type='text/css'>
    @include('css.email')
</style>
```

Sorry for disturbing, I need to test whole situation a bit more...
